typescript/web/src/components/website/Hero/Hero.tsx

```diff
@@ -80,7 +80,7 @@ export function Hero() {
                   px="8"
                   rightIcon={<BsArrowRight />}
                 >
-                  Try it now
+                  Try it! No login required
```

Comment:
This text seems too long to be in a button. Especially since there will also be the "See code on Github" button next to it

Reply:
This will cause layout problems on responsive / mobile etc.

Reply:
it is a bit long but there are 2 states to the button anyway in order it is responsive. It is the way it is recommended on the twitter thread. Github can just be a logo